src/component/Square.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+type SquareProps = {
+  value: string;
+  onClick: () => void;
+};
+
+function Square(props: SquareProps) {
```

Comment:
Should use arrow function to define function component
`const Square: React.FC<SquareProps> = props => {...}`